Dockerfile

```diff
@@ -1,12 +1,9 @@
-FROM cgr.dev/chainguard/node:18
+FROM nginxinc/nginx-unprivileged:1.25
```

Comment:
Skal vi ikke bruke chainguard?

Reply:
Jeg og Emil hadde en lang samtale om den 😄 med chainguard sitt nginx-image har man ikke muligheten til å sette miljøvariabler etter at containeren har startet. Det er et problem fordi miljøvariablene som vi har definert i nais-configen injectes i containeren når den starter opp i podden. Men siden chainguard-imaget er helt strippet for alt, så får man ikke gjennomført variable substitution runtime
```
    {
        "key": "OIDC_AUTH_PROXY",
        "value": "${OIDC_AUTH_PROXY}"
    },
    blir ikke til 
        {
        "key": "OIDC_AUTH_PROXY",
        "value": "https://k9-punsj-oidc-auth-proxy.dev.intern.nav.no"
    },
```

Reply:
de har et åpent issue på det, så det kommer sikkert etterhvert. Hvis vi skulle brukt chainguard i dag måtte vi ha satt miljøvariablene når vi bygger punsj-imaget, og da måtte vi hatt separate images for dev og prod

Reply:
Say no more, fam